app/main.py

```diff
@@ -1,6 +1,37 @@
-def get_weather() -> None:
-    # write your code here
-    pass
+import os
+
+import requests
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+
+
+def get_weather(city: str = "Paris") -> None:
+    api_key = os.getenv("API_KEY")
```

Comment:
api_key should be renamed to API_KEY because it represents a constant. Additionally, it should be declared beneath BASE_URL for better organization.